requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 # unittests
 coverage
+Faker==0.9.2
```

Comment:
Faker does not know about Django, it does not know about the max_length of a field. It is up to us to properly support long enough values. Faker provides a wide range of 'valid' values, we should see if we can accommodate them, unless it is not reasonable, in which case we should change the factory/faker.